.travis.yml

```diff
@@ -5,9 +5,11 @@ python:
 
 install:
   - pip install -e.
-  - pip install pytest pytest-flask lxml requests cssselect
+  - pip install pytest pytest-flask lxml requests cssselect flake8
   - git clone --depth 1 https://github.com/pyvec/pyvo-data pyvocz/pyvo-data
 
-script: py.test test_pyvocz
+script:
+  - py.test test_pyvocz
+  - flake8 --max-line-length=100
```

Comment:
If we want to follow PEP 8, let alone check it automatically, let's not add exceptions. Wrap lines at 79 characters (and docstrings/comments at 72). (I actually consider this more important than a lot of the other rules – it allows me to put several windows side-by-side.)